todoman/ui.py

```diff
@@ -55,6 +55,10 @@ def __init__(self, todo, lists, formatter):
             parent=self,
             edit_text=todo.location
         )
+        self._categories = widgets.ExtendedEdit(
+            parent=self,
+            edit_text=','.join(todo.raw_categories)
```

Comment:
Since `todo.raw_categories` is already a string that contains comma-separated categories, `','.join(..)` is not necessary.